Sources/Storage+Info.swift

```diff
@@ -59,4 +44,26 @@ public extension Storage {
             try remove(forKey: $0.name)
         }
     }
+    
+    func file(forKey key: String) throws -> File {
+        let fileUrl = self.fileUrl(forKey: key)
+        let attributes = try fileManager.attributesOfItem(atPath: fileUrl.path)
+        let modificationDate: Date? = attributes[.modificationDate] as? Date
+        let size: UInt64? = attributes[.size] as? UInt64
+
+        return File(
+            name: key,
+            url: fileUrl,
+            modificationDate:
+            modificationDate,
```

Comment:
Parameter should be in same line

Reply:
f6ec076c554da990c348fa126308d99257553def